articles/azure-sql/virtual-machines/windows/availability-group-distributed-network-name-dnn-listener-configure.md

```diff
@@ -174,7 +174,7 @@ Test the connectivity to your DNN listener with these steps:
 
 ## Port considerations
 
-DNN listeners are designed to listen on all IP addresses, but on a specific, unique port. The DNS entry for the listener name should resolve to the addresses of all replicas in the availability group. This is done automatically with the PowerShell script provided in the [Create Script](#create-script) section. Since DNN listeners accept connections on all IP addresses, it is critical that the listener port be unique, and not in use by any other replica in the availability group. Since SQL Server always listens on port 1433, either directly or via the SQL Browser service, port 1433 cannot be used for any DNN listener.
+DNN listeners are designed to listen on all IP addresses, but on a specific, unique port. The DNS entry for the listener name should resolve to the addresses of all replicas in the availability group. This is done automatically with the PowerShell script provided in the [Create Script](#create-script) section. Since DNN listeners accept connections on all IP addresses, it is critical that the listener port be unique, and not in use by any other replica in the availability group. Since SQL Server by default listens on port 1433, either directly or via the SQL Browser service, it is strongly discouraged to use port 1433 canno for any DNN listener.
```

Comment:
```suggestion
DNN listeners are designed to listen on all IP addresses, but on a specific, unique port. The DNS entry for the listener name should resolve to the addresses of all replicas in the availability group. This is done automatically with the PowerShell script provided in the [Create Script](#create-script) section. Since DNN listeners accept connections on all IP addresses, it is critical that the listener port be unique, and not in use by any other replica in the availability group. Since SQL Server listens on port 1433 by default, either directly or via the SQL Browser service, using port 1433 for the DNN listener is strongly discouraged. 
```

Reply:
That looks good.  ship it!